db/sql.h

```diff
@@ -55,6 +55,8 @@ enum transaction_level {
  * of appsock threads with small stacks. */
 
 #define MAX_HASH_SQL_LENGTH 8192
+#define MAX_USERNAME_LEN 17
+#define MAX_PASSWORD_LEN 19
```

Comment:
Would it be good to add them to cdb2_constants.h and expose via comdb2_limits table?

Reply:
It would. We will address in a subsequent PR.